### ISSUE TITLE
Allow to search for gene symbols instead of Ensembl IDs

### ISSUE THREAD

**Issue by zichner** [thinkh/provenance_retrieval#38]
Currently, only Ensembl IDs are tracked, however, many people would search by gene symbol.

**Comment by mstreit** [comment]
I am sure @thinkh loves this ticket because he raised this multiple times already. :-) The downside is that it is quite hard to fix because it concerns two software modules that don't know each other (by design).

**Comment by thinkh** [comment]
Yes, see also https://github.com/Caleydo/ordino/issues/167. Unfortunately, there is no good solution at the moment that is not a hack. 😐
